starter-project-web-react/cypress/e2e/auth.spec.cy.ts

```diff
@@ -0,0 +1,34 @@
+describe('Authentication', ()=>{
+
+    it("should login user route to home", ()=>{
+        cy.visit('/auth/login')
+    
+        cy.get('[name="email"]').type(Cypress.env('TEST_EMAIL'))
+        cy.get('[name="password"]').type(Cypress.env('TEST_PASSWORD'))
+        cy.contains('Submit').click()
+        
+        cy.wait(10000)
```

Comment:
the wait is never really recommended, if there are other ways to handle it, it is better. For instance in this case, creating a callback after intercepting the API call and the fact that it finishes.

Additionally, since we are just testing the frontend and not the backend, better to mock the API calls in general and work with our own test environment that does not depend on backend database status